code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/NesterovAlexanderTest.java

```diff
@@ -0,0 +1,89 @@
+package ru.unn.agile.ComplexNumber.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class NesterovAlexanderTest {
+    @Test
+    public void isRightNormComplexNumber() {
+        // Arrange
+        ComplexNumber z1 = new ComplexNumber(2, 6);
+        ComplexNumber z1Inv = new ComplexNumber(2, -6);
+        double normZ = Math.pow(2, 2) + Math.pow(6, 2);
+
+        // Act
+        ComplexNumber z = z1.multiply(z1Inv);
+
+        // Assert
+        assertEquals(new ComplexNumber(normZ, 0), z);
+    }
+
+    @Test
+    public void isRightRelativeMathRelation() {
+        // Arrange
+        ComplexNumber z1 = new ComplexNumber(2.22, 4.67);
+        ComplexNumber z1Inv = new ComplexNumber(2.22, -4.67);
+
+        // Act
+        ComplexNumber z = z1.add(z1Inv);
+
+        // Assert
+        assertEquals(4.44, z.getReal() + z.getImaginary(), 1e-10);
+    }
+
+    public double getArg(final ComplexNumber num) {
+        double normNum = Math.sqrt(Math.pow(num.getReal(), 2) + Math.pow(num.getImaginary(), 2));
+        return Math.asin(num.getImaginary() / normNum);
+    }
+
+    @Test
+    public void canArgNumberHalfPI() {
+        // Arrange
+        ComplexNumber z = new ComplexNumber(0, 1);
+
+        // Act
+        double res = getArg(z);
+
+        // Assert
+        assertEquals(res, Math.PI / 2, 1e-15);
+    }
+
+    @Test
+    public void canArgNumberZero() {
```

Comment:
I don't like the way you name test.

In the test you are trying to do the following:

- Create a complex number which arg MUST be 0
- Calculate arg
- check that it is EXACTLY the 0

but your test names as "Can it be zero?" It's better to rename as "argIsZero"

Reply:
I think that this concern is applied also for all of the tests which are based on `arg` logic

Reply:
done